.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -687,11 +687,23 @@ private DateTimeResolutionResult ParseSingleTimePoint(string text, DateObject re
                     ret.Success = true;
                 }
 
-                if (dateContext != null)
+                if (dateContext != null && text != "today")
```

Comment:
Language-specific terms should not be hardcoded in base classes. Please use some appropriate declaration from YAML or declare a new one.
Also, can't a range be "from 05/may to yesterday", for example? Perhaps this can be generalized a little.

Reply:
get it!